common/workunit/workunit.cpp

```diff
@@ -1087,6 +1089,17 @@ class CDaliWorkUnit : public CPersistedWorkUnit
         VStringBuffer path("/GraphProgress/%s/%s", queryWuid(), graphName);
         return querySDS().connect(path, myProcessSession(), RTM_LOCK_WRITE|RTM_CREATE_QUERY, SDS_LOCK_TIMEOUT);
     }
+    IPropertyTree *getGraphProgressTree() const
+    {
+        IRemoteConnection *conn = queryProgressConnection();
```

Comment:
Not new, but if someone keeps calling getGraphProgressTree() or similar that reuses the connection via queryProgressConnection() - it will never see the updates, correct?

That is probably okay, because the IWorkUnit (CDaliWorkUnit), is supposed to be short lived..